connector-webhook/src/main/java/com/redhat/cloud/notifications/connector/webhook/ExchangeProperty.java

```diff
@@ -1,11 +1,15 @@
 package com.redhat.cloud.notifications.connector.webhook;
 
-public class ExchangeProperty {
+public final class ExchangeProperty {
 
     public static final String TARGET_URL_NO_SCHEME = "targetUrlNoScheme";
     public static final String TRUST_ALL = "trustAll";
     public static final String BASIC_AUTH_USERNAME = "basic-auth-username";
     public static final String BASIC_AUTH_PASSWORD = "basic-auth-password";
     public static final String BEARER_TOKEN = "Bearer";
     public static final String INSIGHT_TOKEN_HEADER = "X-Insight-Token";
+    public static final String INCREMENT_ENDPOINT_SERVER_ERRORS = "incrementEndpointServerErrors";
+    public static final String DISABLE_ENDPOINT_CLIENT_ERRORS = "disableEndpointClientErrors";
```

Comment:
Note for later.

Since the disabling of endpoints will eventually be applied to most connectors, it would be great if we used generic terms rather than something tightly coupled to HTTP.

We could have `recoverable-error` (5xx for HTTP, most IOExceptions, etc.) and `unrecoverable-error` (4xx for HTTP, UnknownHostException, etc.), or `temporary-error` and `permanent-error`.

Reply:
Ack